QUEST_LV_0200.tsv

```diff
@@ -6549,50 +6549,50 @@ QUEST_LV_0200_20150406_006552	Defeat orange Lizard Men
-QUEST_LV_0200_20150414_006599	
+QUEST_LV_0200_20150406_006556	Tama Leaf collection
+QUEST_LV_0200_20150406_006557	Investigator Monahan was dispatched by Klaipeda to investigate the damage to the farm. Tell him about the Tama Leaves first.
+QUEST_LV_0200_20150406_006558	You have collected all the Tama Leaves. Please tell Investigator Monahan.
+QUEST_LV_0200_20150406_006559	Investigator Monahan needs more of the earth's aura. Gather some grapevines from the Avarze Vineyard.
+QUEST_LV_0200_20150406_006560	I gathered the necessary grapevines. Please return to Investigator Monahan.
+QUEST_LV_0200_20150406_006561	Collect Orange Treant Horns as evidence.
+QUEST_LV_0200_20150406_006562	Farm Supervisor Dorren said you may enter the farm under the condition of doing some work. Go and eliminate Drakes and gather Orange Treant Horns as proof.
+QUEST_LV_0200_20150406_006563	I collected all the Treant Horns. Please tell Farm Supervisor Dorren.
+QUEST_LV_0200_20150406_006564	%s Treants killed
+QUEST_LV_0200_20150406_006565	Farm Supervisor Dorren will tell you the location the of the Druid if you bring back the seed pouches the soldiers have hidden. Look inside the old wooden chest.
+QUEST_LV_0200_20150406_006566	Elminate the Orange Treants
+QUEST_LV_0200_20150406_006567	Farm Soldier Gedson is hoping to eliminate the Orange Treants. Please rid the farm of Orange Treants.
+QUEST_LV_0200_20150406_006568	You have eliminated all of the Treants. Please inform Farm Soldier Gedson.
+QUEST_LV_0200_20150406_006569	Tenant Charlotte said a large amount of Chupacabras gather at the smell of grape seeds. Please eliminate the Chupacabras that are coming out of the bushes to investigate.
+QUEST_LV_0200_20150406_006570	Eliminate the Cists near the brewery
+QUEST_LV_0200_20150406_006571	Tenant Lewis would like to go work and help the brewery, but the monsters are in the way. Please eliminate the Cists near the Brewery.
+QUEST_LV_0200_20150406_006572	A considerable number of Cists have been defeated. Please return to Tenant Lewis.
+QUEST_LV_0200_20150406_006573	Eliminate the Pink Tree Root Moles
+QUEST_LV_0200_20150406_006574	Baren is worried about the damage to the farm caused by the Carcashus. Reduce the number of Carcashus in the area.
+QUEST_LV_0200_20150406_006575	A number of Carcashus have been eliminated. Return to Baren.
+QUEST_LV_0200_20150406_006576	Eliminate the Pink Tree Root Moles beyond the tunnels
+QUEST_LV_0200_20150406_006577	Eliminate the Glass Mole near the weathervane tower
+QUEST_LV_0200_20150406_006578	Captain Buros tells you that you went too crazy and the Glass Mole has appeared. Go eliminate the Glass Mole by the weathervane tower
+QUEST_LV_0200_20150406_006579	The Glass Mole has been eliminated. Please report to Captain Buros.
+QUEST_LV_0200_20150406_006580	Eliminate Glass Mole
+QUEST_LV_0200_20150414_006581	I want to help purify the thorny forest, but I lack the strength.{nl}Can you eliminate some of the Tree Ghouls bothering the soldiers?
```

Comment:
Is thorny forest a map name? If so, it would be better to capitalize it to Thorny Forest, no?